pkgs/build-support/kernel/make-initrd-ng.nix

```diff
@@ -37,9 +37,9 @@ in
 , extension ? _compressorMeta.extension or
     (throw "Unrecognised compressor ${_compressorName}, please specify filename extension")
 
-# List of { object = path_or_derivation; symlink = "/path"; }
+# List of { source = path_or_derivation; target = "/path"; }
```

Comment:
Source of truth:

https://github.com/NixOS/nixpkgs/blob/8c8f28fa429c6b3d2f0259cee0160135aadcec91/pkgs/build-support/kernel/make-initrd-ng/src/main.rs#L39-L44

Could also mention the `dlopen` config as well.